### DISCUSSION TITLE
Is there an option to force multiline strings on toBe_TODO()?

### DISCUSSION THREAD

**Discussion by reactormonk** [diffplug/selfie#397] [Q&A]
Would be useful for json literals, as they're way easier to read without all the escaped `"`.

**Comment by nedtwigg** [comment]
Good idea. One option is to add a field into `SelfieSettings`, but I think it's better if it just "does the right thing". If somebody later wants it to do something unusual, we'll add a settings field then.

The trick is to encode the strings as multi if they have an `\n` or an `"`, but only for Kotlin and Java post-15. Happy to take a PR for this:

https://github.com/diffplug/selfie/blob/6a70f8f5fe24300cfe20a03956dd4fbd4eb1f2e5/jvm/selfie-lib/src/commonMain/kotlin/com/diffplug/selfie/guts/Literals.kt#L123-L142
